src/systemcmds/topic_listener/generate_listener.py

```diff
@@ -18,74 +18,33 @@
 topics = []
 message_elements = []
 
+# large and not worth printing (find better solution)
+raw_messes = [raw_messages.remove(x) for x in raw_messages if 'qshell_req' in x]
```

Comment:
`raw_messes = [x for x in raw_messes if not any(exception in x for exception in ['qshell_req', 'ulog_stream', ...])]`

Reply:
done